pages/Transfer.qml

```diff
@@ -396,15 +396,16 @@ Rectangle {
       }
 
       RowLayout {
+          Layout.topMargin: 4
           StandardButton {
               id: sendButton
               rightIcon: "qrc:///images/rightArrow.png"
               rightIconInactive: "qrc:///images/rightArrowInactive.png"
-              Layout.topMargin: 4
               text: qsTr("Send") + translationManager.emptyString
               enabled: {
                 updateSendButton()
               }
+              visible: !appWindow.viewOnly && (!appWindow.viewOnly === !persistentSettings.doNotRelayMode)
```

Comment:
Could you do `!(appWindow.viewOnly || persistentSettings.doNotRelayMode)` here or am I missing something?

Reply:
Nice and simple 👍 Thanks